pkg/asset/agent/image/cpio.go

```diff
@@ -68,6 +85,8 @@ func (ca *CpioArchive) StoreFile(filename string) error {
 	if err != nil {
 		return err
 	}
+	header.Name = filepath.Join(dstPath, header.Name)
```

Comment:
nit: I think this should be `path` (always `/` separators) and not `filepath` (separator depends on local OS)?